tests/test_0088-read-with-http.py

```diff
@@ -26,8 +28,6 @@ def test_issue176_again():
 
 @pytest.mark.network
 def test_issue121():
-    with uproot.open(
-        "https://github.com/CoffeaTeam/coffea/raw/master/tests/samples/nano_dy.root"
-    ) as f:
+    with uproot.open(skhep_testdata.data_path("uproot-issue121.root")) as f:
```

Comment:
This one wouldn't count as `@pytest.mark.network` because `scikit-hep_testdata` downloads and caches the file, so Uproot is using the local file system. The reason for the `@pytest.mark.network` is to skip some tests when running on high-latency networks, such as a home computer, because Uproot needs to make several round trips to find seek points within a file. On a home computer, the file that was fetched by `scikit-hep_testdata` is probably already local (we've probably already run this test).